modules/spo-recording-profiles.adoc

```diff
@@ -108,22 +110,22 @@ I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "conta
 +
 [source,terminal]
 ----
-$ oc -n openshift-security-profiles delete pod my-pod
+$ oc -n my-namepace delete pod my-pod
 ----
 
 . Confirm the Security Profiles Operator reconciles the two {type} profiles:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc get {object} -n my-namespace
+$ oc get seccompprofiles -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
 ----
 +
 .Example output
 [source,terminal]
 ----
-NAME                   USAGE                                       STATE
-test-recording-nginx   test-recording-nginx_my-namespace.process   Installed
-test-recording-redis   test-recording-redis_my-namespace.process   Installed
+NAME                   STATUS      AGE
+test-recording-nginx   Installed   56s
+test-recording-redis   Installed   56s
```

Comment:
Again, please do not update the output for selinxprofile when updating output for seccompprofile.
For seccompprofile, the output should be:
```
NAME                   STATUS      AGE
test-recording-nginx   Installed   56s
test-recording-redis   Installed   56s
```
For selinuxporfile, the output should be:
```
NAME                   USAGE                                       STATE
test-recording-nginx   test-recording-nginx_my-namespace.process   Installed
test-recording-redis   test-recording-redis_my-namespace.process   Installed
```

Reply:
Thank you - your instruction is clear. I've created two example output sections.

Reply:
Done, thank you @xiaojiey !